### PR TITLE
Add Difficulty levels

### PR DESCRIPTION
**Pull request by maciek04786**

### CODE REVIEW

File: practice.html [hunk 1]
```diff
@@ -19,6 +19,21 @@ <h2 class="subtitle">Practice your typing skills!</h2>
         <div class="dynamic-text-box">
             <p id="quote"></p>
         </div>
+        <div id="levelSelector">
+                <h3>Choose difficulty level:</h3>
+                <div class="button-group">
+                    <input type="radio" id="levelOne" name="radio-button">
+                    <label for="levelOne" class="radio-label">1</label>
```
Comment (arpitghura) [inline practice.html:26]: Add some more relevent names to the difficulty level. 

like `Beginner`, `intermediate`, `advanced`.

**Review by arpitghura — CHANGES_REQUESTED** [review]
do the required changes.

**Comment by maciek04786** [comment]
Will do. Sorry, I should have thought about that.

On 21 Oct 2022, 07:41, at 07:41, Arpit Ghura ***@***.***> wrote:
***@***.*** requested changes on this pull request.
>
>do the required changes.
>
>> @@ -19,6 +19,21 @@ <h2 class="subtitle">Practice your typing
>skills!</h2>
>         <div class="dynamic-text-box">
>             <p id="quote"></p>
>         </div>
>+        <div id="levelSelector">
>+                <h3>Choose difficulty level:</h3>
>+                <div class="button-group">
>+                    <input type="radio" id="levelOne"
>name="radio-button">
>+                    <label for="levelOne"
>class="radio-label">1</label>
>
>Add some more relevent names to the difficulty level. 
>
>like `Beginner`, `intermediate`, `advanced`.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/arpitghura/typing-test/pull/31#pullrequestreview-1150419726
>You are receiving this because you authored the thread.
>
>Message ID:
***@***.***>

**Review by arpitghura — APPROVED** [review]
You really done a great job.

**Comment by maciek04786** [comment]
Thank you, it's been a pleasure. Happy coding!

On 21 Oct 2022, 18:42, at 18:42, Arpit Ghura ***@***.***> wrote:
***@***.*** approved this pull request.
>
>You really done a great job.
>
>
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/arpitghura/typing-test/pull/31#pullrequestreview-1151334932
>You are receiving this because you authored the thread.
>
>Message ID:
***@***.***>

**Pull request merged**